External/FEXCore/Source/Interface/Core/CPUID.cpp

```diff
@@ -109,6 +109,29 @@ FEXCore::CPUID::FunctionResults CPUIDEmu::Function_01h() {
   return Res;
 }
 
+// 2: Cache and TLB information
+FEXCore::CPUID::FunctionResults CPUIDEmu::Function_02h() {
+  FEXCore::CPUID::FunctionResults Res{};
+
+  Res.eax = 0x1 | // lowest byte always 0x01
+    (0x1 << 8) | // Instruction TLB: 4 KByte pages, 4-way set associative, 32 entries
+    (0x3 << 16) | // Data TLB: 4 KByte pages, 4-way set associative, 64 entries
+    (0x6 << 24);  // L1I: 8K
+
+  Res.ebx = 0xC | // L1D: 16K
+    (0x21 << 8) | // L2: 256K
+    (0x22 << 16) | // L3: 512K
+    (0 << 24);  // null descriptor
+
+  Res.ecx = 0 | // null descriptors
+    (1 << 31); // Invalid data
```

Comment:
This is "reserved for future use" according to the intel manual

Perhaps better to just fill this this register (and edx) with 4 null descriptors (aka, `0x00000000`)

Reply:
This is also what intel do in their example, register ecx is filled with all zeros.

![image](https://user-images.githubusercontent.com/138484/108710471-fa40ad00-7578-11eb-9974-2e1830dbe973.png)

Reply:
The docs I was using say otherwise, but they are far older. I'll update to also return this table, because why not.